plotly/tests/test_core/test_tools/test_Quiver.py

```diff
@@ -0,0 +1,223 @@
+import plotly
+from plotly.graph_objs import *
+import plotly.tools as tls
+from nose.tools import raises
+import numpy as np
+
+
+@raises(Exception)
+def unequal_xy_length():
+    data = tls.Quiver(x=[1, 2], y=[1], u=[1, 2], v=[1, 2])
+
+
+@raises(Exception)
+def unequal_uv_length():
+    data = tls.Quiver(x=[1, 2], y=[1, 3], u=[1], v=[1, 2])
+
+
+@raises(Exception)
+def test_wrong_kwarg():
+    data = tls.Quiver(stuff='not gonna work')
+
+
+def test_one_arrow():
+    nan = np.nan
```

Comment:
you can't have a numpy dependency in our core tests, can you switch these from `plotly/tests/test_core` to `plotly/tests/test_optional`?